ArrtSource/Utils/Loggin.cpp

```diff
@@ -33,48 +33,47 @@ void ArrtAppWindow::LogMessageHandler(QtMsgType type, const QString& category, c
     QListWidgetItem* item = new QListWidgetItem();
     item->setText(line);
 
-    QtMsgType prevIcon = m_maxLogType;
+    int prevIcon = m_maxLogType;
 
     switch (type)
     {
         case QtDebugMsg:
             item->setIcon(QIcon(":/ArrtApplication/Icons/debug.svg"));
             break;
+        case QtInfoMsg:
+            item->setIcon(QIcon(":/ArrtApplication/Icons/info.svg"));
+            m_maxLogType = std::max(m_maxLogType, 1);
+            break;
         case QtWarningMsg:
             item->setIcon(QIcon(":/ArrtApplication/Icons/warning.svg"));
-            m_maxLogType = std::max(m_maxLogType, QtWarningMsg);
+            m_maxLogType = std::max(m_maxLogType, 2);
             break;
```

Comment:
> m_maxLogType, 2);
hmm, this works worse than before. What is the reasoning to use integer here?

Reply:
to fix the message sorting